integrations/criteo/lib/index.js

```diff
@@ -198,21 +204,14 @@ Criteo.prototype.setExtraData = function() {
   var ret = [];
   var extraData = {};
 
-  // Add userId if available as customer_id
-  var userId = this.analytics.user().id();
-
-  // Criteo does NOT want emails passed as customer_id.
-  if (userId && !isEmail(userId)) {
-    ret.push({ event: 'setCustomerId', id: userId });
-  }
-
   // Check cached traits for any that have been defined as extraData params.
   var traits = this.analytics.user().traits();
 
   // Criteo has a special tag for emails.
   // They also require all emails get passed as md5 hashes.
-  if (traits.email) {
-    ret.push({ event: 'setEmail', email: md5(traits.email) });
+  if (traits.email && typeof traits.email === 'string') {
+    var userEmail = traits.email.trim().toLowerCase();
```

Comment:
Despite what Criteo's documentation says, we need to keep the existing functionality of hashing the email "raw" (without trimming or lower casing it. This line will cause duplicated user profiles in Criteo.

Reply:
Understood